pymeasure/instruments/keithley/keithleyDMM6500.py

```diff
@@ -0,0 +1,1432 @@
+        ":SENS:CURR:AC:RANG?",
+        ":SENS:CURR:AC:RANG:AUTO 0;:SENS:CURR:AC:RANG %g",
+        """ A floating point property that controls the AC current range in
+        Amps, which can take values from 0 to 3 A, and 10 A (available for rear terminals).
+        Auto-range is disabled when this property is set. """,
+        validator=truncated_range,
+        values=[0, 10],
+    )
+    current_ac_relative = Instrument.control(
+        ":SENS:CURR:AC:REL?",
+        ":SENS:CURR:AC:REL %g",
+        """ A floating point property that controls the DC current relative value in Amps,
+        which can take values from -3.0 to 3.0 A. When relative offset is enabled, all 
+        subsequent measured readings are offset by the value that is set for this command.
+        If the instrument acquires the value, read this setting to return the value that 
+        was measured internally. """,
+        validator=truncated_range,
+        values=[-3.0, 3.0],
+    )
+    current_ac_relative_status = Instrument.control(
+        ":SENS:CURR:AC:REL:STAT?",
+        ":SENS:CURR:AC:REL:STAT %g",
+        """ A property queries, enables or disables the application of a relative offset value 
+        to the measurement. Takes string :code:`on|True|1` or :code:`off|False|0`. """,
+        validator=strict_discrete_set,
+        values={"on": 1, "off": 0, True: 1, False: 0, 1: 1, 0: 0},
+        map_values=True,
+    )
+    current_ac_nplc = Instrument.control(
+        ":SENS:CURR:AC:NPLC?",
+        ":SENS:CURR:AC:NPLC %g",
+        """ A floating point property that controls the number of power line cycles
+        (NPLC) for the AC current measurements, which sets the integration period
+        and measurement speed. Takes values from 0.01 to 10, where 0.1, 1, and 10 are
+        Fast, Medium, and Slow respectively. """,
+    )
+    current_ac_digits = Instrument.control(
+        ":DISP:CURR:AC:DIG?",
+        ":DISP:CURR:AC:DIG %d",
+        """ An integer property that determines the number of digits in the AC current
+        readings, which can take values from 3 to 6 representing dispaly digits from 3.5 to 6.5.""",
+        validator=truncated_discrete_set,
+        values=[4, 5, 6, 7],
+        cast=int,
+    )
+    current_ac_bandwidth = Instrument.control(
+        ":SENS:CURR:AC:DET:BAND?",
+        ":SENS:CURR:AC:DET:BAND %g",
+        """ A floating point property that sets the AC current detector
+        bandwidth in Hz, which can take the values 3, 30, and 300 Hz. """,
+        validator=truncated_discrete_set,
+        values=[3, 30, 300],
+    )
+
+    def measure_current(self, max_current=10e-3, ac=False):
+        """Configures the instrument to measure current,
+        based on a maximum current to set the range, and
+        a boolean flag to determine if DC or AC is required.
+
+        :param max_current: A current in Volts to set the current range
+        :param ac: False for DC current, and True for AC current
+        """
```

Comment:
For coding style (Python recommendation) see the [documentation](https://pymeasure.readthedocs.io/en/latest/dev/coding_standards.html#docstrings).

Reply:
All functions' docstring were rewritten. If there's anything need to change the style, please let me know. Thank you.